tfhe/src/boolean/parameters/mod.rs

```diff
@@ -45,8 +44,8 @@ pub struct BooleanParameters {
     pub lwe_dimension: LweDimension,
     pub glwe_dimension: GlweDimension,
     pub polynomial_size: PolynomialSize,
-    pub lwe_modular_std_dev: StandardDev,
-    pub glwe_modular_std_dev: StandardDev,
+    pub lwe_noise_distribution: DynamicDistribution<u32>,
+    pub glwe_noise_distribution: DynamicDistribution<u32>,
```

Comment:
TUniform checks if the required boud_log2 is valid for the given integer type and is generic over the integer type (via a phantom data), Boolean using u32 instead of u64 it's adapted to the boolean case :)